cmd/bosun/sched/sched.go

```diff
@@ -589,13 +589,6 @@ func (s *Schedule) Run() error {
 		go s.PingHosts()
 	}
 	go s.Poll()
-	go func() {
-		// Sleep before starting unknown checks. If Bosun was down for a little
-		// while then all the state.Touched times will be old and a bunch of
-		// unknowns will go off because it hasn't run all the checks yet.
-		time.Sleep(s.Conf.CheckFrequency)
```

Comment:
@captncraig The feature Matt means is delaying the unknown check at startup. Basically we want to give Bosun time to collect queued data before firing off a bunch of unknowns.